client-src/elements/chromedash-feature-page.ts

```diff
@@ -421,6 +422,48 @@ export class ChromedashFeaturePage extends LitElement {
     });
   }
 
+  handlePlaceOnHold() {
+    if (
+      !confirm(
+        'Place this feature on hold?  It will not appear on the roadmap.'
+      )
+    )
+      return;
```

Comment:
`{ return; }` parenthesis around the return;?

Reply:
Added